apps/alpha-test-app/src/App.vue

```diff
@@ -81,5 +89,11 @@ const toggleDark = useToggle(isDark);
     </template>
 
     <RouterView />
+
+    <OnyxColorSchemeDialog
+      v-model="colorScheme"
+      :open="isColorSchemeDialogOpen"
```

Comment:
We could do this, but it would differ from the `OnyxDialog` API then 🤔

Reply:
maybe do the same for OnyxDialog? when the prop and emit control the same ref in the end, it very much feels like a perfect use-case for vmodel?

Reply:
But the v-model will always only emit false since the dialog does not open itself :/ Also I think not forcing it to be a boolean is quiet helpful for cases like this which might be very common:

```ts
const itemIdToEdit = ref(3);

<OnyxDialog :open="itemIdToEdit != undefined" @close="itemIdToEdit = undefined" />
```

Reply:
ah right, oki doki :)